iot_verifier/src/poc.rs

```diff
@@ -404,7 +404,9 @@ fn verify_entropy(
     entropy_end: DateTime<Utc>,
     received_ts: DateTime<Utc>,
 ) -> GenericVerifyResult {
-    if received_ts < entropy_start || received_ts > entropy_end {
+    if received_ts.timestamp() < entropy_start.timestamp()
```

Comment:
I don't understand the need for this change

Reply:
This is not strictly necessary but as the source timestamps of entropy are in seconds precision, its better fitting to compare the receive timestamp at the same precision